community_supplied/Kernel/Multicast/check-multicast-iif-mismatch-error-cnt-iagent.rule

```diff
@@ -0,0 +1,103 @@
+/*
+ * This rule checks monitors the health of the system on the basis of the iif_mismatch_err_cnt parameter.
+ *
+ * The sensor used is iAgent.
+ * The rule shows red alarm when the iif_mismatch_err_cnt exceeds 100.
+ * It shows yellow alarm when the iif_mismatch_err_cnt is in the range 10-100.
+ */
+
+iceberg {
+    topic kernel.multicast {
+        rule check-multicast-iif-mismatch-error-cnt-iagent {
+            sensor iif-mismatch-sensor {
+                synopsis iif-mismatch-sensor;
+                description "iAgent sensor for IIF mismatch error count";
+                iAgent {
+                    file iif_mismatch_cnt.yml;
+                    table iifMismatchErrCountTable;
+                    frequency 10m;
+                }
+            }
+            field multicast-iif-mismatch-error-cnt {
+                sensor iif-mismatch-sensor {
+                    path iif_mismatch_err_cnt;
+                }
+                type integer;
+                description "Multicast IIF Mismatch Error Count";
+            }
+            trigger report-multicast-iif-mismatch-iagent {
+                synopsis "Trigger for reporting the iif mismatch error count, with iAgent sensor";
+                description "This trigger helps in monitoring the health of the system with the help of the parameter iif mismatch error count. If this value is increasing atmost by a value of 10, the system is healthy. Else if it is increasing between the range 10-100, it is a warning sign. Else, the system is under risk";
+                frequency 10m;
+                term iif-mismatch-err-cnt-diff-gt-100 {
+                    when {
+                        not-equal-to "$multicast-iif-mismatch-error-cnt" 0;
+                        increasing-at-least-by-value "$multicast-iif-mismatch-error-cnt" {
+                            value 101;
+                            time-range 1300s;
+                            any;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "Multicast IIF mismatch error count difference between successive samples is greater than 100. Urgent action is required.";
```

Comment:
It is good to display actual error count in message. Modify all rule's trigger messages